### DISCUSSION TITLE
dp train problem

### DISCUSSION THREAD

**Discussion by maverick422** [deepmodeling/deepmd-kit#1343] [Q&A] [answered]
when i running a dp train  with a examples input file  
it goes 

KeyError: 'key `systems` is required in arguments but not found'

here is my input file 
[input.json.txt](https://github.com/deepmodeling/deepmd-kit/files/7675975/input.json.txt)

with a deepmd-kit-1.3.3-cuda10.0_gpu-Linux-x86_64

**Comment by njzjz** [comment, marked as answer]
Hi, you need to use the example from your version (1.3.3) instead of the latest version.

**Reply by maverick422** [reply]
thanks I will give it  try！
